### ISSUE TITLE
Date-time format problems. Plexos output issue?

### ISSUE THREAD

**Issue by tdeetjen** [NREL/rplexos#44]
I'm having a problem where the .db file created via process_folder() is incorrectly translating datetime data from Plexos. 
In the Plexos results viewer, a piece of generation data might be tagged with 1/5/2014. 
After using process_folder(), this data will be stored in the .db file with the time stamp 1/4/2014 00:00:00. (As if 00:00:00 is the last hour of the day). 

However, whenever I open the \R\win-library\3.2\rplexos\extdata\solution .zip file in Plexos, and use the process_folder() command to create a .db from it, the date 3/14/2015 is correctly transferred to the .db as 3/14/2015 00:00:00.

Could you share the XML Plexos input file you used to calculate the .zip solution file? I'm guessing there are some settings in Plexos that are affecting its data output, so I would like to see how you have yours set up.

**Comment by eibanez** [comment]
Data is here: https://github.com/NREL/rplexos/tree/master/inst/extdata

I'm not actively developing the package so I cannot offer any assistance.

**Comment by tdeetjen** [comment]
Do you have the load-timedata.csv and wind-timedata.csv needed to run the
simulation?

Thanks for responding - if I find a solution I'll put it on the github bug
page.

*Thomas Deetjen*
Mechanical Engineering
The University of Texas at Austin
Webber Energy Group
<http://www.webberenergygroup.com/people/thomas-deetjen/>
tdeetjen@gmail.com

**Comment by eibanez** [comment]
No

**Comment by tdeetjen** [comment]
Thanks, Eduardo. I found a solution.

I compared my Horizon and Reports setup with what you had in the three_nodes.xml Plexos input file and made the following changes to my Horizon and Report settings to match yours:
Horizon:
Chrono Step Type: Day --> Hour
Look Ahead Type: Day(s) --> Interval(s)
Look Ahead Periods per Day: 6 --> 24
Report:
Output Results by Day: No --> Yes
Datetime Convention: End of Period --> Beginning of Period

The rplexos package worked correctly after making these changes. 

I haven't tested which of these changes is responsible for correcting the problem, but it is probably due to changing the Report>DatetimeConvention to "Beginning of Period". Perhaps this should be included in reference manual as many Plexos users likely organize their reports using the "End of Period" convention which is the common form for real-time market data.
